Sources/MobileSdk/BLEConnection.swift

```diff
@@ -0,0 +1,178 @@
+// Derived from MIT-licensed work by Paul Wilkinson: https://github.com/paulw11/L2Cap
+
+import CoreBluetooth
+import Foundation
+
+/// The base BLE connection, only intended for subclassing.
+class BLEInternalConnection: NSObject, StreamDelegate {
```

Comment:
This is only used for L2CAP right? Should it be renamed to be clearer?

Reply:
Possibly; I'll see if I can find a better name.

Reply:
It is now `BLEInternalL2CAPConnection`.